src/azure-cli/azure/cli/command_modules/acs/managed_cluster_decorator.py

```diff
@@ -4706,17 +4706,13 @@ def get_k8s_support_plan(self) -> Union[str, None]:
 
         :return: string or None
         """
-        # default to None
-        support_plan = None
-        # try to read the property value corresponding to the parameter from the `mc` object
-        if self.mc and hasattr(self.mc, "support_plan") and self.mc.support_plan is not None:
-            support_plan = self.mc.support_plan
-
-        # if specified by customer, use the specified value
+        # take input
```

Comment:
please also add some live test cases (see examples in test_aks_commands.py) for this option (set/update k8s support plan)

Reply:
I am aware of that, maybe I can add it later. 

maintaining live test is costly, you need to keep maintain the recording file, well this unittest have almost no maintance cost.